lib/plugin-loader.js

```diff
@@ -959,12 +971,66 @@ PluginLoader.prototype = {
               bootstrapLogger.severe(e.message);
               resolve(); // We don't want to reject here. Error gets caught down stream
             });
+          })
+        }).then(() => { /* Obtains and stores the endpoints exposed by the agent */
```

Comment:
Think this comment was meant to be on line 945, line 975 can say the same thing except for API ML